lib/connect/api/errors.js

```diff
@@ -16,8 +16,48 @@ class HttpError extends Error {
    */
   constructor(status, message) {
     super(message);
+    /**
+     * The http status code.
+     * @type {number}
+     */
     this.status = status;
   }
 }
-
-module.exports = { HttpError };
+class APIError extends HttpError {
+/**
+ * APIError wraps any http response which status is != 2xx.
+ *
+ * Provides a quick and easy access to the JSON error.
+ *
+ * The Cloud Blue Connect JSON error looks like:
+ *
+ * @example
+ * {
+ *    "error_code": "SYS_001",
+ *    "errors": [
+ *       "error message 1"
+ *    ]
+ * }
+ *
+ *
+ */
+  constructor(status, message) {
+    super(status, message);
+    /**
+     * The JSON error object.
+     * @type {Object}
+     */
+    this.json = JSON.parse(message);
```

Comment:
[MINOR] For me `json` is too abstract maybe we can follow ES here and name it `toJSON`